packages/aws-cdk-lib/aws-lambda/test/function.test.ts

```diff
@@ -3220,32 +3328,6 @@ test('function using a reserved environment variable', () => {
   })).toThrow(/AWS_REGION environment variable is reserved/);
 });
 
-test('set SnapStart to desired value', () => {
```

Comment:
Why are we removing this test?

Reply:
Sorry for the confusion, I just moved this test with the new added snapstart test into the snapstart section.

Reply:
Ahh I see it now. Thanks for pointing that out!